draft-packages/loading-spinner/KaizenDraft/LoadingSpinner/LoadingSpinner.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from "react"
+import colorTokens from "@kaizen/design-tokens/tokens/color.json"
+import spacingTokens from "@kaizen/design-tokens/tokens/spacing.json"
+const styles = require("./styles.module.scss")
+
+type size = "sm" | "md"
+
+export interface LoadingSpinnerProps {
+  accessibilityLabel: string
+  /**
+   * Generally use "md" unless spinner is inside a form field
+   * @default "md"
+   */
+  size: size
```

Comment:
You've specified a default size but the prop is still required, probs want to make this optional?